apis/network/v1alpha3/zone_types.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	runtimev1alpha1 "github.com/crossplane/crossplane-runtime/apis/core/v1alpha1"
+)
+
+// +kubebuilder:object:root=true
+
+// Zone is a managed resource that represents an AWS Route53 Hosted Zone.
+// +kubebuilder:printcolumn:name="ID",type="string",JSONPath=".status.atProvider.Id"
```

Comment:
```suggestion
// +kubebuilder:printcolumn:name="ID",type="string",JSONPath=".metadata.annotations['crossplane.io/external-name']"
```

should work but didn't test manually

Reply:
I tested it and it doesn't seem to be working.